cadence/contracts/KittyItems.cdc

```diff
@@ -270,6 +271,21 @@ pub contract KittyItems: NonFungibleToken {
 
             KittyItems.totalSupply = KittyItems.totalSupply + (1 as UInt64)
         }
+
+        // Update NFT images for new type
+        pub fun addNewImagesForKind(from: AuthAccount, newImages: {Kind: {Rarity: String}}) {
```

Comment:
Method was added to the minter Resource to ensure that only minters are able to update images listed in the contract.